src/main/scala/firrtl/transforms/DedupAnnotations.scala

```diff
@@ -6,7 +6,7 @@ package transforms
 import firrtl.ir._
 import firrtl.Mappers._
 import firrtl.options.Dependency
-import firrtl.Utils.BoolType
+import firrtl.Utils._
```

Comment:
```suggestion
import firrtl.Utils.{BoolType, groupByIntoSeq}
```
Not enough importing to justify wildcard just yet